cli

```diff
@@ -1,39 +1,17 @@
 #!/usr/bin/env node
```

Comment:
Why did you decide to rename this file? I don't see how it is related to the task.

Reply:
I've split this file to the server and cli. This is done bc cli triggers actions such as migrate or compile, which should be done regardless of ts compilation. Otherwise we would have cli actions both in src and dist, both in ts and js, which is not convenient at all.